### PR TITLE
add initiation time, total_rate to Call and init go modules

### LINKED ISSUE #28: The "total_rate" field is not exposed in the Call struct
The Call struct does not expose a field for accessing the `total_rate`,  which is returned by the REST API (as documented [here](https://www.plivo.com/docs/voice/getting-started/advanced/get-details-of-a-single-call/)). 
I was hoping to access this field after retrieving a `Call` instance from the `CallService.Get` method.

Is it possible to have this field added to the struct?

### PR DESCRIPTION
**Pull request by oliver006**
This adds initial_time and total_rate to the Call object (fixes #28) and adds support for go modules.

I also ran `go fmt` on the package.

### CODE REVIEW

**Comment by oliver006** [comment]
@kunal-plivo done

Any chance you can also look at some of the bug fix PRs like #37 / #63 and #62 ?

File: go.sum [hunk 1]
```diff
@@ -0,0 +1,18 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
```
Comment (kunal-plivo) [inline go.sum]: can we please delete this file as well

Comment (oliver006) [reply go.sum]: My bad, removed it.

Comment (oliver006) [reply go.sum]: And the commits.

**Comment by oliver006** [comment]
@kunal-plivo - any chance this can be revisited and merged?

**Comment by kunal-plivo** [comment]
Sorry @oliver006, I forgot to approve these changes
@nixonsam can we please take this up in next release

**Pull request closed without merging**